Testing/VelaptorTests/Content/ContentLoaderTests.cs

```diff
@@ -74,10 +68,10 @@ public void Load_WhenLoadingAtlasData_LoadsAtlasData()
             var loader = CreateContentLoader();
 
```

Comment:
Refactor method to match method to test
```suggestion
public void LoadAtlas_WhenLoadingAtlasData_LoadsAtlasData()
```